src/contexts/SessionContext.jsx

```diff
@@ -57,7 +57,7 @@ export const SessionProvider = ({
 
   useEffect(() => {
     if (onSessionRestore !== undefined) {
-      onSessionRestoreClient(onSessionRestore);
+      session.events.on(EVENTS.SESSION_RESTORED, onSessionRestore);
```

Comment:
Fixes for Inrupt's v2 library upgrade. `onSessionRestoreClient` and all other `onEvent` functions have been removed and replaced with this `session.events.on` style.